primitives/src/ceremonies.rs

```diff
@@ -161,10 +158,10 @@ impl<Signature, AccountId: Clone + Encode, Moment: Encode + Copy>
 			.encode()
 	}
 
-	#[cfg(feature = "std")]
+	#[cfg(any(feature = "std", feature = "full_crypto"))]
 	pub fn sign<P>(self, pair: &P) -> Self
```

Comment:
Not needed now actually, but might be useful, if we ever migrate to the enclave ;)